app/models/user.rb

```diff
@@ -1,12 +1,24 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] In this activity we want the application asks for **confirmation** of email of users during registration. To achieve, this kindly add the `:confirmable` option to the devise authentication parameters.